components/match2/commons/starcraft_starcraft2/match_summary_starcraft.lua

```diff
@@ -84,16 +84,43 @@ function StarcraftMatchSummary.Header(props)
 			showFlag = false,
 			showLink = opponent.type == 'team',
 		})
-			:addClass('brkts-popup-header-opponent')
 	end
 
-	local header = html.create('div'):addClass('brkts-popup-header-dev')
+	local headerLeft = html.create('div')
+		:addClass('brkts-popup-header-opponent')
+		:addClass('brkts-popup-header-opponent-left')
+		:css('width', '50%')
 		:node(renderOpponent(1))
+		:node(StarcraftMatchSummary._createScore(
+			match.opponents[1],
+			'left',
+			match.showScore
+		))
+	local headerRight = html.create('div')
+		:addClass('brkts-popup-header-opponent')
+		:addClass('brkts-popup-header-opponent-right')
+		:css('width', '50%')
+		:node(StarcraftMatchSummary._createScore(
+			match.opponents[2],
+			'right',
+			match.showScore
```

Comment:
Would it be possible to move `props.match.showScore` to `props.options.showScore`? I feel that showScore is a display option and not a match property since the match can be displayed in other places without the score.

Reply:
with the changes to #716 we even need to change it^^